app-doc/sphinx-rtd-dark-mode/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=sphinx-rtd-dark-mode
+PKGSEC=python
+PKGDEP="python-3 sphinx-rtd-theme"
+BUILDDEP="python-build python-installer setuptools-scm wheel"
+PKGDES="Adds a toggleable dark mode to the Read the Docs theme"
+
+NOPYTHON2=1
+ABSPLITDBG=0
```

Comment:
This is an architecture-agnostic package, replace this line with:

```
ABHOST=noarch
```

To indicate as such.